Newspack/Newspack/Models/StoryFolder+CoreDataClass.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+import CoreData
+
+@objc(StoryFolder)
+public class StoryFolder: NSManagedObject {
+
+    @nonobjc public class func defaultFetchRequest() -> NSFetchRequest<StoryFolder> {
+        return NSFetchRequest<StoryFolder>(entityName: "StoryFolder")
```

Comment:
Same as `fetchRequest()`. **But** I've noticed all of the NSMO concrete classes also have this same duality (defaultFetchRequest / fetchRequest).

Is the goal to safeguard against Xcode dropping the autogenerated method?

Reply:
No. It's to deal with this:
<img width="567" alt="Screen Shot 2020-04-22 at 8 40 53 AM" src="https://user-images.githubusercontent.com/1435271/79989163-2aa94d80-8475-11ea-9bb0-2c241bb07684.png">
The boiler plate `fetchRequest` method conflicts with the `fetchRequest` that's part of the class. Using it yields warnings:
<img width="266" alt="Screen Shot 2020-04-22 at 8 41 09 AM" src="https://user-images.githubusercontent.com/1435271/79989350-5af0ec00-8475-11ea-82bd-fd72a17ef002.png">
Maybe there's a better way to deal with it, but so far I've just been adding a `defaultFetchRequest` method in the class. (I like the naming better too.)